schema/spacewalk/upgrade/susemanager-schema-4.2.10-to-susemanager-schema-4.2.11/158-alma-amazon.sql

```diff
@@ -0,0 +1,21 @@
+insert into rhnPackageProvider (id, name)
```

Comment:
As the schema was already tagged, the migration directory is wrong. It should be in

susemanager-schema-4.2.11-to-susemanager-schema-4.2.12

When doing this it would be good to adapt the dependency of the java source code to the DB schema by changing
`java/tito.props`

```
[dbschema]
conf/rhn_java.conf=4.2.12
```

Reply:
Fixed